cdap-docs/tools/github-release-notes.py

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#  Copyright © 2017 Cask Data, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# GitHub Release Notes
+#
+# Give a CDAP release notes file in rst format, generates a file with release notes for
+# the top-most section in GitHub format (Markdown).
+#
+# Reads and writes using common-sense defaults that can be over-written if required.
+# Default output ends up as 'target/github-release-notes.txt'.
```

Comment:
No... because it is .txt, then the browser will render it as-is. If it is `.md`, the broswer will instead download the file.

As someone has to access it and copy and paste it into the GitHub form, I wanted to make it accessible from within a web browser.